Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM coturn/coturn:4.6.2-r12
```

Comment:
#1 should be merged, and the CI should be extended in this PR to cover PR and push as well.